spec/fixtures/package.json

```diff
@@ -21,8 +21,7 @@
     "noflo-css": "^0.0.2"
   },
   "devDependencies": {
-    "noflo-runtime-iframe": "^0.8.0",
```

Comment:
Should just recheck our examples then, once the noflo-browser build using this is live?